.streamlit/config.toml

```diff
@@ -4,4 +4,4 @@ developmentMode = false
 [server]
 maxMessageSize = 5000
 maxUploadSize = 5000
-port = 8501
\ No newline at end of file
+port = 8502
```

Comment:
number should be 8501

Reply:
Happy to update it but we will need to open a ticket with IT I think. 8502 is already open and linked to:

abi-services.de/flashviewer/
abi-services.de/flashtaggerviewer/

Reply:
Nevermind, you are right. I just pushed!